test/kata1.fizzBuzz.test.js

```diff
@@ -1,11 +1,26 @@
 const { fizzBuzz } = require("../src");
 
 describe("fizzBuzz", () => {
-  test("returns Fizz when passed a multiple of 3", () => {});
+  it("returns Fizz when passed a multiple of 3", () => {
+  const result = fizzBuzz(3)
 
-  test("returns Buzz when passed a multiple of 5", () => {});
+  expect(fizzBuzz(3)).toBe('Fizz')
```

Comment:
I would add a few more cases in here like `expect(fizzBuzz(6)).toBe('Fizz')` and `expect(fizzBuzz(9)).toBe('Fizz')` because otherwise your function might only be returning 'Fizz' for the input argument of 3. It's always to have at least 2 different arguments for each test case so we know things aren't hardcoded.